### DISCUSSION TITLE
mytheme 2

### DISCUSSION THREAD

**Discussion by C2TouteBote** [mathoudebine/turing-smart-screen-python#610] [Themes]
Hello mise a jour de mes thèmes turzx.
[My Themes.zip](https://github.com/user-attachments/files/17969352/My.Themes.zip)
![Capture d'écran 2024-12-01 160447](https://github.com/user-attachments/assets/4f0f0eca-a67a-4d1e-8911-faee8af7c14c)
![Capture d'écran 2024-12-01 160447](https://github.com/user-attachments/assets/9022c538-c357-4fe6-aa71-166ab780057e)
![Capture d'écran 2024-12-01 160504](https://github.com/user-attachments/assets/d63a70b9-01b1-4a84-8f04-d212df530857)
![Capture d'écran 2024-12-01 160511](https://github.com/user-attachments/assets/240633ed-5199-45e4-93a2-6923c8403358)
![Capture d'écran 2024-12-01 160519](https://github.com/user-attachments/assets/5680e68c-7cc8-4f10-8ff8-b334df157f79)
![Capture d'écran 2024-12-01 160526](https://github.com/user-attachments/assets/1269401e-8e0e-4229-9d4f-29fc3138499d)
![Capture d'écran 2024-12-01 160535](https://github.com/user-attachments/assets/0eeac0b5-e44f-43d2-9e68-cc80f899df04)
![Capture d'écran 2024-12-01 160542](https://github.com/user-attachments/assets/3ef434a7-9703-4a8e-9f88-6323cb6b5d07)
![Capture d'écran 2024-12-01 160552](https://github.com/user-attachments/assets/3d266e43-45e8-4afb-89ab-5ccb5dbe8975)

**Comment by xZodiak97** [comment]
![GIGABYTE_theme_example2](https://github.com/user-attachments/assets/0661c904-8d47-4e5b-a513-78cd2397b4c2)
Hello, I was inspired by your theme and I would like to ask if it could be changed to the way I have it, see photo. Just CPU Ryzen 5 7600 and GPU RX 7900XT.

**Reply by C2TouteBote** [reply]
oui vous pouvez

**Comment by Jaxeman09** [comment]
how do we use this??
isnt it supposed to .yaml files?
